ophyd/device.py

```diff
@@ -560,7 +560,7 @@ def stage(self) -> List[object]:
                 stage_sigs[k] = v
 
         # Read current values, to be restored by unstage()
-        original_vals = {sig: sig.get() for sig, _ in stage_sigs.items()}
+        original_vals = {sig: sig.get() for sig in stage_sigs.keys()}
```

Comment:
In fact, we don't even need `.keys()` here!

Reply:
haha yes!